backend/Bindings/Edna.Bindings.LtiAdvantage/LtiAdvantageBindingExtensions.cs

```diff
@@ -16,7 +17,7 @@ public static IWebJobsBuilder AddLtiAdvantageBindings(this IWebJobsBuilder build
         {
             builder.AddExtension<LtiAdvantageExtensionConfigProvider>();
 
-            builder.Services.AddHttpClient();
+            builder.Services.AddHttpClient(nameof(EdnaExternalHttpHandler)).ConfigurePrimaryHttpMessageHandler(() => new EdnaExternalHttpHandler());

```

Comment:
We've added a new line in file **Lti1BindingExtensions.cs** for doing this? Any specific reason for not being consistent across these files?

Reply:
I will make this consistent.